src/components/Pressable/utils.ts

```diff
@@ -119,12 +119,80 @@ const adaptTouchEvent = (event: GestureTouchEvent): PressableEvent => {
   };
 };
 
+type StylePropKeys = (keyof ViewStyle)[];
+
+const innerStyleKeys = [
+  'alignContent',
+  'alignItems',
+  'flex',
+  'flexBasis',
+  'flexDirection',
+  'flexWrap',
+  'rowGap',
+  'gap',
+  'columnGap',
+  'justifyContent',
+  'overflow',
+  'padding',
+  'paddingBottom',
+  'paddingEnd',
+  'paddingHorizontal',
+  'paddingLeft',
+  'paddingRight',
+  'paddingStart',
+  'paddingTop',
+  'paddingVertical',
+  'start',
+  'end',
+
+  /**
+   * @platform ios
+   */
+  'direction',
+] as StylePropKeys;
+
+const extractStyles = (from: ViewStyle, keys: StylePropKeys): ViewStyle => {
+  if (!from) {
+    return {} as ViewStyle;
+  }
+
+  const extractedData = {} as Record<string, unknown>;
+
+  for (const key of keys) {
+    if (from[key] !== undefined) {
+      extractedData[key] = from[key];
+    }
+  }
+
+  return extractedData as ViewStyle;
+};
+
+const excludeStyles = (from: ViewStyle, keys: StylePropKeys): ViewStyle => {
+  if (!from) {
+    return {} as ViewStyle;
+  }
+
+  const exclusiveData = { ...from };
+
+  for (const key of keys) {
+    if (exclusiveData[key] !== undefined) {
+      exclusiveData[key] = undefined;
```

Comment:
Unfortunately I couldn't use `delete` even though it would clear up debugging, due to eslint rule which forbids deleting dynamically computed keys:
![image](https://github.com/software-mansion/react-native-gesture-handler/assets/74246391/5d5a2ae5-ebbe-4b42-90d0-b379e003e69c)

Reply:
In that case I'd consider adding `eslint-disable`. I'm not a fan of assigning `undefined` to CSS styles as it happens now (unless I've missed something) cc @j-piasecki

Reply:
I don't think this is relevant anymore as it's changed to the functional approach.

Reply:
Partially, we still have assigning `undefined` in `outerStyles`.

The thing is, I'm not 100% sure if passing `undefined` into styles will break something or not. I did get an error once, but it mentioned `translationX` so it was probably due to something else (though I got it in `Pressable` example). If we are sure that it doesn't break anything then I'm fine with that 😅

Reply:
The error you're talking about originated from a bad argument being passed to an `Animated` function which we don't use anywhere in `Pressable`, I think it was likely caused by a different component.